### ISSUE TITLE
Fail to download the ucec proteomics data from bcm

### ISSUE THREAD

**Issue by Sunmile** [PayneLab/cptac#57]
Hello,
Thanks for the work in the CPTAC!
I just came across an error.
I can't load the ucec bcm proteomics data with the following code
```
import cptac
ucec = cptac.Ucec()
ucec.get_proteomics('bcm')
```
It will report an error that:
```
[Errno 2] No such file or directory: '/.../anaconda3/lib/python3.11/site-packages/cptac/data/bcm-ucec/UCEC_proteomics_gene_abundance_log2_reference_intensity_normalized_Tumor.txt.gz'
```
Other cancer types all work without error.
The 'umich' proteomics ucec also can be loaded without error. 

Thank you for your attention.
Summile

**Comment by viabard** [comment]
This is the case for me as well!

**Comment by bm600** [comment]
Hello, sorry for the delay in responding. This error should now be resolved with v1.5.14. Please let us know if you are still having issues. Thank you for using the package!
